### ISSUE TITLE
Dashboard failed requests - status code 500 - version 1.0.3

### ISSUE THREAD

**Issue by jdbolts** [laravel/horizon#137]
I have horizon installed and each page seems to work as it should apart from the dashboard.

The below image shows that the 2 metrics (workload and stats) are constantly loading,
![dashboard_capture](https://user-images.githubusercontent.com/8181433/30328050-202cf918-97c6-11e7-9119-9161f0e5cf34.PNG)

I have checked the console and the error i am getting is below:

Workload Error:

    {message: "Parse error: syntax error, unexpected '='",…}
        exception : "Symfony\Component\Debug\Exception\FatalThrowableError"
        file : "/var/www/html/tsn/vendor/laravel/horizon/src/Repositories/RedisWorkloadRepository.php"
        line : 70
        message : "Parse error: syntax error, unexpected '='"
        trace : [{file: "/var/www/html/tsn/vendor/composer/ClassLoader.php", line: 301,…},…]

Stats error:

    {message: "Parse error: syntax error, unexpected '='",…}
        exception : "Symfony\Component\Debug\Exception\FatalThrowableError"
        file : "/var/www/html/tsn/vendor/laravel/horizon/src/WaitTimeCalculator.php"
        line : 75
        message : "Parse error: syntax error, unexpected '='"
        trace : [{file: "/var/www/html/tsn/vendor/composer/ClassLoader.php", line: 301,…},…]

Any help would be appreciated.

**Comment by themsaid** [comment]
Looks like you're running an old version of PHP, make sure you're on PHP7.0 or later.

**Comment by jdbolts** [comment]
Doing php -v on the server shows that the php version is 7.1.9 so surely this cant be the issue

**Comment by themsaid** [comment]
Check your php-fpm version, this command informs you with the version of php-cli

**Comment by rossuhms** [comment]
@jdbolts I had this issue and reported it a few weeks ago. It did turn out to be the incorrect version of PHP was running.  See issue #114 

When I was running `php -v` I was showing version 7.1.8 for the CLI. I completely forgot to check php-fpm, so when I ran phpinfo() I was in fact only running PHP 7.0. 

I corrected this by running `brew unlink php70 && brew uninstall php70` to remove the old version and then ran `brew install homebrew/php/php71`to install the correct version of PHP. I then restarted Laravel Valet and Horizon worked as it should.

:)
